hindi/bollywood.md

```diff
@@ -6,4 +6,6 @@ SRK ,[go here](actor1/srk.md)
 
 Salman,[go here](actor2/salman.md)
 
-Amithab, [go here](actor3/amithab.md)
\ No newline at end of file
+Amithab, [go here](actor3/amithab.md)
+
+आमिर खान तो भूल ही गए ये लोग! [यहाँ दबाये] (actor4/aamir.mdamir.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.